plugins/config/repository.yaml

```diff
@@ -24,6 +24,116 @@ repositories:
 # wildcards are allowed in name, entries are processed in top-down order
 # first entry matching the name is used
 projects:
+    # Use nightlies in all main branches
+    - name: gazebo-cmake3
```

Comment:
These should all be `gz`

Reply:
[6189aee](https://github.com/ignition-tooling/gzdev/pull/57/commits/6189aee4ea36daee7b259078a68f80f65142783c)